controllers/reconcile_configmap.go

```diff
@@ -19,6 +19,13 @@ const ConfigMapName = "config"
 const ConfigDataField = "config"
 
 func reconcileConfigMap(ctx context.Context, c client.Client, log logr.Logger, namespace string) error {
+	existing := &corev1.ConfigMap{}
+	err := c.Get(ctx, types.NamespacedName{Namespace: namespace, Name: ConfigMapName}, existing)
+	if k8serrors.IsNotFound(err) {
```

Comment:
Will this work in a scenario where:
- We create an addresspool CR
- We create later the metallb CR

(I believe so, as the configmap will trigger another reconcile of this, but can we add a test?)

Reply:
done, added a new test to cover this scenario.